autorest/codegen/models/parameter.py

```diff
@@ -319,6 +324,10 @@ def is_hidden(self) -> bool:
             self.yaml_data["implementation"] == "Client" and self.constant
         )
 
+    @property
+    def is_content_type(self) -> bool:
+        return self.serialized_name == "content_type" and self.location.value == "header"
```

Comment:
Can you switch this to `self.rest_api_name == "Content-Type"`? It's really the header name that we care about more, rather than the serialized name.

Also can just be `self.location == ParameterLocation.Header`

Reply:
updated, thanks for the specific code